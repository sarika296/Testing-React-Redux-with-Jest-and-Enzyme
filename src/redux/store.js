import { createStore, applyMiddleware } from 'redux'
import thunk from 'redux-thunk'
import { composeWithDevTools } from 'redux-devtools-extension'
import Reducer from './reducer/Reducer'
import logger from 'redux-logger'

const store= createStore(Reducer, composeWithDevTools(applyMiddleware(logger, thunk)) )

export default store 