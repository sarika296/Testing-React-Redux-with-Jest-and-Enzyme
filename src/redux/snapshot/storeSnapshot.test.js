import React from 'react'
import renderer from 'react-test-renderer'
import store from '../store'

describe('Store Snapshot Testing...', () => {

    it('renders store component correctly', () => {
        const tree = renderer.create(<store />).toJSON();
        expect(tree).toMatchSnapshot();
    });

}); 
