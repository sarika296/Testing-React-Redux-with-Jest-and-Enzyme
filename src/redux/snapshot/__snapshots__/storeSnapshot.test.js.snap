// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Store Snapshot Testing... renders store component correctly 1`] = `<store />`;
